pyfar/dsp/dsp.py

```diff
@@ -780,6 +785,17 @@ def regularized_spectrum_inversion(
             numerical aspects of linear inversion. Philadelphia: SIAM, 1998.
 
     """
+    # Check frequency range parameter
+    if frequency_range is None and freq_range is None:
+        raise ValueError("Frequency range must be provided")
```

Comment:
```suggestion
    if frequency_range is None and freq_range is None:
        raise ValueError("Frequency range must be provided")
```
this is double, since it qould throw an error in line 813 for both cases.